runtimes/pythonrt/pythonrt.go

```diff
@@ -485,6 +489,17 @@ func (py *pySvc) initialCall(ctx context.Context, funcName string, args []sdktyp
 	}
 }
 
+func (py *pySvc) drainPrints(ctx context.Context) {
```

Comment:
Please add a docstring explaining this function's purpose: it's easy to understand what it does, but not why.

Reply:
this is go, not py :) so you mean comment.

Reply:
Done